src/Components/Shared/src/WebEventData.cs

```diff
@@ -244,5 +273,28 @@ private static ChangeEventArgs DeserializeChangeEventArgs(string eventArgsJson)
             }
             return changeArgs;
         }
+
+#nullable disable
+        // WebView has different nullability settings compared to Server and WebAssembly
+        // which weirds out JSON's nullability for these types. Disable nullability for this contract
+        // until we can update everything to haave uniform nullability.
```

Comment:
That's unexpected and awkward. Is it just a mistake we made, or do you think there's a reason we wanted it to be different?

Reply:
I think it might have been an oversight - the WebView2 project have nullable=annotations. My hope is to make all source projects have nullable=enabled in p6: https://github.com/dotnet/aspnetcore/pull/32742.